app/src/main/java/com/example/theperiodpurse/ui/setting/SettingScreenLayout.kt

```diff
@@ -0,0 +1,179 @@
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.example.theperiodpurse.R
+
+
+@Composable
+fun SettingScreenLayout(
+    modifier: Modifier = Modifier,
+    onNotificationClicked: () -> Unit,
+    onBackUpClicked: () -> Unit,
+    onDeleteClicked: () -> Unit,
+){
+   Column(modifier = modifier
+       .fillMaxSize()
+       .verticalScroll(rememberScrollState())) {
+       Text(
+           text = stringResource(R.string.tracking_preferences),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+
+       TrackingPreferencesRow()
+       Text(
+           text = stringResource(R.string.notifications_heading),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+       Row(modifier = modifier.padding(20.dp)) {
+           var checked by remember { mutableStateOf(false) }
+           Column (modifier = Modifier) {
+               Text(text = stringResource(
+                   R.string.remind_me_to_log_symptoms),
+                   fontWeight = FontWeight.Bold)
+               Spacer(modifier = modifier.padding(3.dp))
+               Text(text = stringResource(R.string.sample_notification))
+           }
+           Switch(
+               checked = checked,
+               onCheckedChange = { checked = !checked},
+               modifier = modifier
+                   .fillMaxWidth()
+                   .wrapContentWidth(Alignment.End),
+               colors = SwitchDefaults.colors(
+                   uncheckedThumbColor = Color.DarkGray
+               )
+           )
+       }
+      TabOption(
+          stringResource(id = R.string.customize_notifications),
+          onTabClicked = onNotificationClicked
+      )
+       Text(
+           text = stringResource(R.string.account_settings_heading),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+       TabOption(
+           text = stringResource(R.string.back_up_account),
+           onTabClicked = onBackUpClicked
+       )
+       Spacer(modifier = Modifier.padding(10.dp))
+       TabOption(
+           text = stringResource(id = R.string.delete_account),
+           onTabClicked = onDeleteClicked
+       )
+       Spacer(modifier = Modifier.padding(10.dp))
+       Text(text = "The Period Purse. All rights reserved.")
```

Comment:
perhaps use a string resource here instead? Looks like something we will reuse in other places